packages/ui-admin/src/workspace/bots/CreateBotModal.tsx

```diff
@@ -208,13 +230,62 @@ class CreateBotModal extends Component<Props, State> {
             {this.state.categories.length > 0 && (
               <FormGroup label={lang.tr('admin.workspace.bots.create.category')}>
                 <Select
-                  tabIndex="4"
                   options={this.state.categories}
                   value={this.state.selectedCategory}
                   onChange={selectedCategory => this.setState({ selectedCategory: selectedCategory as any })}
                 />
               </FormGroup>
             )}
+            <FormGroup
+              label={lang.tr('admin.workspace.bots.create.cloud')}
+              labelFor="checkbox-bot-cloud"
+              helperText={lang.tr('admin.workspace.bots.create.cloudHelper')}
+            >
+              <Checkbox
+                id="checkbox-bot-cloud"
+                label={lang.tr('admin.workspace.bots.create.cloudCheckbox')}
+                checked={this.state.isCloudBot}
+                onChange={e =>
+                  this.setState({
+                    isCloudBot: e.target.checked
+                  })
+                }
+              />
+            </FormGroup>
+            {this.state.isCloudBot && (
+              <FormGroup
+                label={lang.tr('admin.workspace.bots.create.cloudConfiguration')}
+                labelFor="cloud-client-id"
+                helperText={lang.tr('admin.workspace.bots.create.cloudConfigurationHelper')}
+              >
+                <InputGroup
+                  id="cloud-client-id"
+                  placeholder={lang.tr('admin.workspace.bots.create.clientIdPlaceholder')}
+                  value={this.state.cloudClientId}
+                  className={style.clientId}
+                  pattern="\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b"
```

Comment:
I think we could avoid specific patterns at this point, what I would suggest is just accepting anything on this page, then on the "Config" of the studio page we can put a whole section to explain everything correctly, and actually make a call to validate that specified api/key are valid